dspace-api/src/main/java/org/dspace/mock/MetadataValueDTO.java

```diff
@@ -5,11 +5,12 @@
  *
  * http://www.dspace.org/license/
  */
-package org.dspace.content.crosswalk;
+package org.dspace.mock;
```

Comment:
Can we please change this package name too?  The word `mock` is again misleading. It could be changed to simply `org.dspace.content.dto`, or even just `org.dspace.dto` is fine too.  Either is better in my opinion.